raiden_contracts/data/gas.json

```diff
@@ -4,20 +4,21 @@
     "CustomToken.transfer": 36123,
     "CustomToken.transferFrom": 29557,
     "MonitoringService.claimReward": 41942,
-    "MonitoringService.monitor": 207777,
+    "MonitoringService.monitor": 207760,
+    "OneToN.bulkClaim": 81653,
```

Comment:
Measure at least two options.  Maybe `OneToN.bulkClaim 1 claim` and `OneToN.bulkClaim 6 claims`.